### ISSUE TITLE
Add dataset: yalt_ai_tabular_dataset

### ISSUE THREAD

**Issue by davanstrien** [bigscience-workshop/lam#67]
### A URL for this dataset

https://zenodo.org/record/6827706

### Dataset description

>This dataset has been built to train a segmentation model. It contains ALTO and YOLOv5 formats

> This dataset is derived from the LECTAUREP Dataset

### Dataset modality

Image

### Dataset licence

Creative Commons Attribution 4.0 International

### How can you access this data

As a download from a repository/website

### Confirm the dataset has an open licence

- [X] To the best of my knowledge, this dataset is accessible via an open licence

**Comment by davanstrien** [comment]
#self-assign

**Comment by davanstrien** [comment]
#ready-for-review

**Comment by davanstrien** [comment]
I've implemented this dataset via a loading script: https://huggingface.co/datasets/biglam/yalt_ai_tabular_dataset.
The original dataset includes a YOLO formatted version. I've included two configs:

- YOLO
- COCO

I included the COCO config because, at the moment, some libraries prefer/expect this format, including the transformers object detection feature extraction classes (e.g. Detr) https://github.com/huggingface/transformers/blob/f0d496828d3da3bf1e3c8fbed394d7847e839fa6/src/transformers/models/detr/feature_extraction_detr.py#L122. 

The COCO config does some conversion to get the YOLO annotations into a COCO-compatible format. There is some COCO information that can't be inferred from this format. Still, this conversion means that hopefully, most libraries expecting COCO will not complain since the required keys expected in the COCO annotations dictionary are present. 

@NielsRogge sorry to ping you directly, but I just wondered if it was likely that the `feature_extractors` will also support other input formats, e.g. YOLO? There are likely to be a few object detection datasets added as part of this hackathon, so I want to ensure I'm targeting a helpful format (from the transformers library perspective). 

@albertvillanova @mariosasko, if either of you has a quick chance to look at this, that would be amazing. I didn't find many examples of datasets for object detection in the hub, so I want to make sure this looks like a reasonable approach, particularly since we'll probably be adding more of these datasets in the coming weeks.
